src/components/__tests__/InfiniteHits.js

```diff
@@ -118,6 +154,43 @@ it('renders correctly on the last page', () => {
   expect(wrapper.html()).toMatchSnapshot();
 });
 
+it('renders correctly when not on the first page', () => {
+  __setState({
+    ...defaultState,
+    isFirstPage: false,
+    isLastPage: false,
+  });
+
+  const wrapper = mount(InfiniteHits, {
+    propsData: {
+      showPrevious: true,
+    },
+  });
+
+  expect(wrapper.html()).toMatchSnapshot();
```

Comment:
We could add an assertion on the `button` to help understand what we actually test.